src/registry/routes/helpers/apply-default-values.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var _ = require('underscore');
+
+module.exports = function(requestParameters, expectedParameters) {
+  var result = _.clone(requestParameters);
```

Comment:
is this `clone` necessary? If a component has a decent number of params (like TMS) this clone could use some CPU. If not necessary, I would be tempted to remove it? What do you think